### ISSUE TITLE
Lanelet2 has gtest warnings in ROS2 Galactic

### ISSUE THREAD

**Issue by h-ohta** [fzi-forschungszentrum-informatik/Lanelet2#209]
Warnings is as follows:
```
In file included from /opt/ros/galactic/src/gtest_vendor/include/gtest/gtest.h:71,
                 from 
.../lanelet2/lanelet2_routing/test/test_routing_graph_container.cpp:1:
/opt/ros/galactic/src/gtest_vendor/include/gtest/gtest-typed-test.h:227:64: warning: ‘constexpr bool testing::internal::TypedTestCaseIsDeprecated()’ is deprecated: TYPED_TEST_CASE is deprecated, please use TYPED_TEST_SUITE [-Wdeprecated-declarations]
  227 |   static_assert(::testing::internal::TypedTestCaseIsDeprecated(), ""); \
```
this warnings occurs in many places throughout buildings.

### My Environment
OS: Ubuntu20.04
ROS: ROS2 Galactic

### Expected Behaviros
No warnings

**Comment by h-ohta** [comment]
I make PR https://github.com/fzi-forschungszentrum-informatik/Lanelet2/pull/210

**Comment by h-ohta** [comment]
This issue is fixed In 0c52870.

**Comment by poggenhans** [comment]
Thank you anyways. We had the fix already underway when you created this PR.
